### DISCUSSION TITLE
Spark dependencies cron is needed for...?

### DISCUSSION THREAD

**Discussion by tingeltangelthomas** [jaegertracing/jaeger#4349] [Q&A]
Hi all,
can somebody explain me, why jaeger in prod environment needs the spark-dependencies cron?

> This is a Spark job that collects spans from storage, analyze links between services, and stores them for later presentation in the UI. Note that it is needed for the production deployment. all-in-one distribution does not need this job.

In my mind I connect this job only with the dependencies map on the ui. I don't need this map at all.

Thank you!!

**Comment by yurishkuro** [comment]
The job is not needed if you're not interested in the service map. But I think the tab will remain in the UI, I don't think we have a config option to turn it off (easy to add though).
